### ISSUE TITLE
relabel() doesn't respect cols()

### ISSUE THREAD

**Issue by basnijholt** [ioam/holoviews#146]
This works:
```python
(im * hv.VLine(0) + im.sample(x=0)
+im * hv.HLine(0) + im.sample(y=0)).relabel("Dirac cone").cols(2)
```
this not

```python
(im * hv.VLine(0) + im.sample(x=0)
+im * hv.HLine(0) + im.sample(y=0)).cols(2).relabel("Dirac cone")
````

In the second case ```cols(2)``` is not applied and the layout just contains four images side by side.

**Comment by jlstevens** [comment]
Thanks for filing this.. should be easy to fix and we will have a look at it later.

**Comment by jlstevens** [comment]
Should be fixed in e12f084
